app/Http/Controllers/MembersController.php

```diff
@@ -282,19 +282,10 @@ public function searchSkills(Request $request)
 
 		$all_skills = Skill::pluck('tag', 'id')->toArray();
 
-		// TODO I think this can be even shorter when defining a $num_matches or something based on $require_how
-		switch ($require_how) {
-			case 'any':
-				$members = Member::whereHas('skills', function (Builder $query) use ($skills, $require_how) {
-					$query->whereIn('id', $skills);
-				})->get();
-				break;
-			case 'all':
-				$members = Member::whereHas('skills', function (Builder $query) use ($skills, $require_how) {
-					$query->whereIn('id', $skills);
-				}, '=', count($skills))->get();
-				break;
-		}
+		$num_matches = ($require_how == 'all') ? count($skills) : 1;
+		$members = Member::whereHas('skills', function (Builder $query) use ($skills, $require_how) {
+			$query->whereIn('id', $skills);
+		}, '>=', $num_matches)->get();
```

Comment:
This is great.
If you need to also have the skills in the view use `with('skills')`